sklearn/metrics/tests/test_score_objects.py

```diff
@@ -18,7 +18,7 @@
 from sklearn.base import BaseEstimator
 from sklearn.metrics import (f1_score, r2_score, roc_auc_score, fbeta_score,
                              log_loss, precision_score, recall_score)
-from sklearn.metrics.cluster import adjusted_rand_score
+from sklearn.metrics import cluster as cluster_module
```

Comment:
I prefer you export directly all the necessary metrics as its done on the line before.